TreeGrid.php

```diff
@@ -244,6 +248,9 @@ public function renderTableRow($model, $key, $index)
 
         $parentId = ArrayHelper::getValue($model, $this->parentColumnName);
         if ($parentId) {
+            if('collapsed' == ArrayHelper::getValue($this->pluginOptions, 'initialState')){
```

Comment:
Okay, I will modify it. Thank you.  
Effect picture  
![Effect picture](http://git.oschina.net/uploads/images/2015/1126/182731_706d030d_5211.png "Effect picture")